config/jobs/kubernetes-sigs/image-builder/image-builder-ova-presubmits.yaml

```diff
@@ -0,0 +1,99 @@
+presets:
+  - labels:
+      preset-cluster-api-provider-vsphere-e2e-config: "true"
+    env:
+      - name: JUMPHOST
+        valueFrom:
+          secretKeyRef:
+            name: clusterapi-provider-vsphere-ci-prow
+            key: jumphost
+      - name: GOVC_URL
+        valueFrom:
+          secretKeyRef:
+            name: clusterapi-provider-vsphere-ci-prow
+            key: vsphere-server
+      - name: GOVC_USERNAME
+        valueFrom:
+          secretKeyRef:
+            name: clusterapi-provider-vsphere-ci-prow
+            key: vsphere-username
+      - name: GOVC_PASSWORD
+        valueFrom:
+          secretKeyRef:
+            name: clusterapi-provider-vsphere-ci-prow
+            key: vsphere-password
+      - name: TARGET_VM_SSH
+        valueFrom:
+          secretKeyRef:
+            name: clusterapi-provider-vsphere-ci-prow
+            key: target-vm-ssh
+      - name: TARGET_VM_SSH_PUB
+        valueFrom:
+          secretKeyRef:
+            name: clusterapi-provider-vsphere-ci-prow
+            key: target-vm-ssh-pub
+    volumeMounts:
+      - name: jumphost-key
+        mountPath: /root/ssh/.jumphost
+      - name: bootstrapper-key
+        mountPath: /root/ssh/.bootstrapper
+      - name: vpn-conf
+        mountPath: /root/.openvpn
+    volumes:
+      - name: jumphost-key
+        secret:
+          secretName: clusterapi-provider-vsphere-ci-prow
+          defaultMode: 256
+          items:
+            - key: jumphost-key
+              path: jumphost-key
+      - name: bootstrapper-key
+        secret:
+          secretName: clusterapi-provider-vsphere-ci-prow
+          defaultMode: 256
+          items:
+            - key: bootstrapper-key
+              path: bootstrapper-key
+      - name: vpn-conf
+        secret:
+          secretName: cluster-api-provider-vsphere-vpn-config
+          defaultMode: 256
+          items:
+            - key: prow.ovpn
+              path: prow.ovpn
+            - key: client.crt
+              path: client.crt
+            - key: client.key
+              path: client.key
+            - key: ca.crt
+              path: ca.crt
+            - key: tls.key
+              path: tls.key
+
+presubmits:
+  kubernetes-sigs/image-builder:
+    - name: pull-ova-all
+      labels:
+        preset-dind-enabled: "true"
+        preset-kind-volume-mounts: "true"
+        preset-cluster-api-provider-vsphere-e2e-config: "true"
+      run_if_changed: 'images/capi/.*'
```

Comment:
this job should be disabled by default and optional while we test https://github.com/kubernetes-sigs/image-builder/pull/425 with it. Once https://github.com/kubernetes-sigs/image-builder/pull/425 merges, we can update it to run on all PRs that touch capi/

Reply:
see https://github.com/kubernetes/test-infra/pull/18532/files#r463098679

Reply:
makes sense. Thanks.

Reply:
you need to remove `run_if_changed` and add `optional: true`

Reply:
Duh. Done.